### DISCUSSION TITLE
Connecting to a PostgreSQL database using AWS

### DISCUSSION THREAD

**Discussion by Crhysus** [Budibase/budibase#8706] [Help]
Hi I'm very new to this. I have created an Amazon Web Server RDS database and connected it to PGAdmin and created a table with my data but am unable to connect to database through BudiBase.
<img width="675" alt="image" src="https://user-images.githubusercontent.com/118404076/202319350-3484eb99-d790-4eff-b8b3-19941519d99a.png">
If anyone has any idea where I'm going wrong some help would be appreciated. 
Thanks.

**Comment by drearystate** [comment]
I am having the same issue. I have asked questions on Discord as well with no responses. I am using bit.io as my postgresql database and I keep getting a "invalid credentials" message. My database is set as public and I need SSL turned on but i think its something as simple as I am filling out the "Ca" field wrong.

![image](https://user-images.githubusercontent.com/20757700/202921131-20a8ad7d-5100-447f-9381-5ae21146ad2d.png)

**Comment by melohagan** [comment]
Hey @Crhysus @drearystate 

I think you do need to fill out the **Ca** field. The CA should be a largish text dump. Something akin to:

```
-----BEGIN CERTIFICATE-----
MIIC3jCCAcYCCQD6vfS16kRWSDANBgkqhkiG9w0BAQsFADAxMQswCQYDVQQGEwJV
nEXdLOYrhqs5GD1DGhpXufi4hPDwjrGIQ41A+b2NSmDE31tpgLZVQjHriqinB0tV
HOd+QtzHowfZPGi6DV4SKhzn
-----END CERTIFICATE-----
```

The body will be larger, I've just truncated a lot of this one, not that it's anything important.

This guide may be of use: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL.Concepts.General.SSL.html
However I have never personally setup Postgres with SSL in AWS.
